### ISSUE TITLE
Huge space between context menu items

### ISSUE THREAD

**Issue by prasadpatil25** [micheleg/dash-to-dock#560]
Hi,

Please check images for Dash to dock context menu and Nautilus context menu.

Should have consistent spacing between menu items so that it will look pretty and same all over the places. Nautilus context menu looks good in space management. Dash to Dock should implement same. 

The height of separator StWidget in Gnome-Shell. Please reduce red highlighted area and make it more consistent with Nautilus context menu.

![dash to dock and nautilus context menus](https://user-images.githubusercontent.com/12842912/29137339-9e0953f0-7d5d-11e7-92d4-9d0436ca2722.png)

**Comment by micheleg** [comment]
Yes, I agree some of the popup menu are a bit fat. The style and use of the separators is  inherited from the shell theme, and not meant to match the gtk style. What I was planning on doing is removing was removing the unnecessary separators. I actually had an experimental branch where I was plying with it:

https://github.com/micheleg/dash-to-dock/tree/experimental_remove_popupmenu_spacer
